### PR TITLE
[SPARK-19827][R][FOLLOWUP] spark.ml R API for PIC

### PR DESCRIPTION
**Pull request by srowen**
## What changes were proposed in this pull request?

Follow up style fixes to PIC in R; see #23072

## How was this patch tested?

Existing tests.

### CODE REVIEW

**Comment by srowen** [comment]
CC @HyukjinKwon @felixcheung @huaxingao

File: R/pkg/R/mllib_fpm.R [hunk 1]
```diff
@@ -183,8 +183,8 @@ setMethod("write.ml", signature(object = "FPGrowthModel", path = "character"),
 #' @return A complete set of frequent sequential patterns in the input sequences of itemsets.
 #'         The returned \code{SparkDataFrame} contains columns of sequence and corresponding
 #'         frequency. The schema of it will be:
-#'         \code{sequence: ArrayType(ArrayType(T))} (T is the item type)
-#'         \code{freq: Long}
+#'         \code{sequence: ArrayType(ArrayType(T))}, \code{freq: integer}
+#'         where T is the item type
```
Comment (huaxingao) [inline R/pkg/R/mllib_fpm.R]: I know there is a problem here too. But the one that needs to get fixed is in line 637 in R/pkg/R/mllib_clustering.R. 
I guess I will have a separate follow up PR to fix the mllib_fpm.R problem. Sorry for the mess.

Comment (srowen) [reply R/pkg/R/mllib_fpm.R]: Yeah I wanted to fix both here while we're at it.

**Review by HyukjinKwon — APPROVED** [review]

File: R/pkg/R/mllib_fpm.R (hunk 1 above)
Comment (felixcheung) [reply R/pkg/R/mllib_fpm.R]: let's fix both

**Review by felixcheung — APPROVED** [review]
LGTM thx @srowen

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**